app/controllers/api/v1x0/portfolio_items_controller.rb

```diff
@@ -72,7 +72,7 @@ def portfolio_item_params
       end
 
       def portfolio_item_patch_params
-        params.permit(:favorite, :name, :description, :orphan, :state, :display_name, :long_description, :distributor, :documentation_url, :support_url, :workflow_ref)
+        params.permit(:favorite, :name, :description, :orphan, :state, :display_name, :long_description, :distributor, :documentation_url, :support_url, :workflow_ref, :id, :service_offering_source_ref)
```

Comment:
@syncrou This is also going to go away and come via the manageiq-api-common from openapi son